docs/contributing.md

```diff
@@ -4,9 +4,38 @@ title: Contribute to Stryker4s
 custom_edit_url: https://github.com/stryker-mutator/stryker4s/edit/master/docs/contributing.md
 sidebar_position: 3
 ---
-
-This is the contribution guide for Stryker4s. Great to have you here! Here are a few ways you can help to make this project better.
-
+## Developing on stryker4jvm:
+Currently, stryker4jvm is using 3 build tools for all its modules. Stryker4jvm-core is managed by maven,
+stryker4jvm-mutator-kotlin is managed by gradle and all other modules are managed by sbt. In order to build/test locally
+you need to take the following steps:
+
+Build has only been confirmed to work with:
+* Jvm 15.0.2,
+* Maven 3.6.3,
+* Kotlin 1.4.20,
```

Comment:
The current latest release of Kotlin is 1.8.10. I'd expect Stryker4jvm to always be tested against the latest (or LTS) version. So, this would really have to be updated before this can be merged (same with the other versions). Scala also isn't mentioned here, but I would suggest not keeping this list, as it will probably rapidly become out of date

Reply:
Stryker4k only seems to work with kotlin 1.4.20, this is a limitation in the original project as well